docsite/rst/intro_windows.rst

```diff
@@ -242,6 +289,100 @@ Looking at an Ansible checkout, copy the `examples/scripts/upgrade_to_ps3.ps1 <h
 
 .. _what_windows_modules_are_available:
 
+Verifying Windows Configuration
+```````````````````````````````
+
+After you have set up the Windows host you can use the following commands to verify the setup to ensure Ansible can communite with the host.
+
+Verify WinRM Listeners
+++++++++++++++++++++++
+
+To see what listeners are currently active you can run ``winrm enumerate winrm/config/listener`` the following command in PowerShell::
+
+   #!powershell
+   PS C:\> winrm enumerate winrm/config/listener
+
+   Listener
+       Address = *
+       Transport = HTTP
+       Port = 5985
+       Hostname
+       Enabled = true
+       URLPrefix = wsman
+       CertificateThumbprint
+       ListeningOn = 127.0.0.1, ::1
+
+   Listener
+       Address = *
+       Transport = HTTPS
+       Port = 5986
+       Hostname = WINDOWS-SERVER
+       Enabled = true
+       URLPrefix = wsman
+       CertificateThumbprint = 1234567890ABCDEF111213141516171819101A1B
+       ListeningOn = 127.0.0.1, ::1
+
+In the above example we can see both a HTTP and HTTPS listener has been created and are listening on ports 5985 and 5986 respectively.
+
+Verify Service Settings
++++++++++++++++++++++++
+
+Once you have verified that WinRM is setup and listener on a port you can use ``winrm get winrm/config/service`` to get the configuration of the WinRM service::
+
+   #!powershell
+   PS C:\> winrm get winrm/config/service
+
+   Service
+       RootSDDL = O:NSG:BAD:P(A;;GA;;;BA)(A;;GR;;;IU)S:P(AU;FA;GA;;;WD)(AU;SA;GXGW;;;WD)
+       MaxConcurrentOperations = 4294967295
+       MaxConcurrentOperationsPerUser = 1500
+       EnumerationTimeoutms = 240000
+       MaxConnections = 300
+       MaxPacketRetrievalTimeSeconds = 120
+       AllowUnencrypted = false
+       Auth
+           Basic = true
+           Kerberos = true
+           Negotiate = true
+           Certificate = false
+           CredSSP = false
+           CbtHardeningLevel = Relaxed
+       DefaultPorts
+           HTTP = 5985
+           HTTPS = 5986
+       IPv4Filter = *
+       IPv6Filter = *
+       EnableCompatibilityHttpListener = false
+       EnableCompatibilityHttpsListener = false
+       CertificateThumbprint
+       AllowRemoteAccess = true
+
+In the above example we are able to authenticate with the Windows host using
```

Comment:
(missing ':' )
> 
In the above example we are able to authenticate with the Windows host using

->

> 
In the above example we are able to authenticate with the Windows host using: